lib/notifications/client/response_precompiled_letter.rb

```diff
@@ -0,0 +1,15 @@
+module Notifications
+  class Client
+    class ResponsePrecompiledLetter
```

Comment:
I'm on the fence about this to be honest. Are there any issues with using `ResponseNotification`? From what I understand, it would leave `content`, `template` and `uri` set to `nil`, which is suboptimal, but I feel like a pre-compiled letter response is still a notification response semantically, since both `id` and `reference` refer to a notification.

Reply:
👍Updated to use `ReponseNotification`, which keeps the code simpler